src/main/java/application/RequestModel.java

```diff
@@ -1,31 +1,17 @@
 package application;
 
-import java.util.Map;
-
 /**
  * A data structure representing a request passed through an InputBoundary Interface.
  */
-public class RequestModel {
-    private final Map<String, Object> data;
-    private final OutputBoundary respondTo;
-
+public abstract class RequestModel {
```

Comment:
I believe that this line can be deleted since it doesn't appear to be an abstract class.